sdks/how-to/how-to-manage-notifications.md

```diff
@@ -4,3 +4,181 @@ description: Learn how to manage notifications
 tags:
 - Notification
 ---
+
+# How to Manage Notifications
+
+## What Is a Notification?
+
+A Notification represent a request from a [Data Owner](/sdks/glossary#data-owner) to a Healthcare Professional to perform 
+an operation.  
+As for now, there are three types of Notifications:
+
+* **KEY_PAIR_UPDATE**: when a Patient loses their private key and gets a new one, they can send this type of Notification to a Healthcare Professional to ask them to share their data with them again.
+* **NEW_USER_OWN_DATA_ACCESS**: when a Patient logs in for the first time, they can send this type of Notification to a Healthcare Professional to aks for access to their own data.
+* **OTHER**: all the other use cases.
+
+:::note
+
+To perform the following operations, we suppose you have at least a Patient and a Healthcare Professional in your database.
+
+:::
+
+## Creating a Notification
+
+In the following example, a Patient creates a Notification for a Healthcare Professional communicating that they have a new
+ key and need access to their data.
+
+<!-- file://code-samples/how-to/manage-notifications/index.mts snippet:create a notification as patient-->
+```typescript
+const accessNotification = await patientApi.notificationApi.createOrModifyNotification(
+  new Notification({
+    type: NotificationTypeEnum.KEY_PAIR_UPDATE,
+  }),
+  hcp.id,
+)
+```
+
+:::note
+
+The default status of a Notification is `pending`
+
+:::
```

Comment:
I think it would be better to have a small section about the lifecycle of a notification:
Initially a notification will be `pending`, when an hcp sees it and starts processing will be `ongoing`, ...

Reply:
I'll add an explanation with a diagram